src/nvlink/worker_32bit.py

```
@@ -0,0 +1,377 @@
+                                if hasattr(data, 'tobytes'):
+                                    # memoryview
+                                    data_bytes = bytes(data)
+                                elif isinstance(data, bytes):
+                                    data_bytes = data
+                                else:
+                                    data_bytes = str(data).encode('utf-8')
+
+                                # Decode data
+                                try:
+                                    data_str = data_bytes[:ret_code].decode('shift-jis', errors='replace')
+                                except:
```

Ruff (0.14.14)

[error] Do not use bare `except`

(E722)